SPECS/util-linux/pam.d/runuser

```diff
@@ -0,0 +1,5 @@
+#%PAM-1.0
```

Comment:
Can our tooling still build/find the sources with this subdirectory? And even if it does, could we move these files directly under `SPECS/util-linux` for the sake of consistency? Or under `SPECS/util-linux/SOURCES` to be aligned with what our documentation for building packages says?

Reply:
As far as I know the tools will search for all files recursively inside the directory regardless of their location.

I was following the layout of the shadow-utils package which includes them in a `pam.d` subdirectory, but I can certainly drop them in the root directory.